lib/builder/webpack/client.js

```diff
@@ -164,6 +144,27 @@ export default class WebpackClientConfig extends WebpackBaseConfig {
       }
     }
 
+    // Make uglifyjs faster
+    if (!this.options.dev && !config.optimization.minimizer) {
+      // https://github.com/webpack-contrib/uglifyjs-webpack-plugin
+      const sourceMap = config.devtool === '#source-map'
```

Comment:
Should the value be source-map?
https://webpack.js.org/configuration/devtool/

Reply:
Well, i`m not really sure here.  
https://github.com/webpack/webpack/blob/master/lib/WebpackOptionsApply.js#L236
Seems like it affect only a comment string for a source map and it will work with # @ and without any of it.